pylabnet/gui/widgets/shutterwidget.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+
+"""
+This file contains wrappers for Jupyter Notebook Widgets used for controling the SC20 Shutter
+
+"""
+
+import ipywidgets as widgets
+
+
+class ShutterToogle():
+    """
```

Comment:
Looks very good! One small comment.
No leading newline is needed in doc-string: a one-sentence short description should start on the same line with quotes.
Then an empty line. Then full detailed description on the following lines.
However, the final quotes should be on a separate line just like you do.